package.json

```diff
@@ -44,22 +44,22 @@
     "primeng": "^11.0.0",
```

Comment:
PrimeNG should be update to 12 for Angular 12.

Reply:
Yes, done, thank you.